====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
height 1029 width 1156
pic should be PIL Image or ndarray. Got <class 'tuple'>
====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
height 1029 width 1156
pic should be PIL Image or ndarray. Got <class 'tuple'>
====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
Traceback (most recent call last):
  File "/home/alexzuzow/Desktop/agar_multiagent/train_DQN.py", line 126, in <module>
    state = agar1.reset()
  File "/home/alexzuzow/Desktop/agar_multiagent/agar.py", line 90, in reset
    self.reset()
  File "/home/alexzuzow/Desktop/agar_multiagent/agar.py", line 90, in reset
    self.reset()
  File "/home/alexzuzow/Desktop/agar_multiagent/agar.py", line 61, in reset
    self.driver.get("https://agar.io/#ffa")
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 436, in get
    self.execute(Command.GET, {'url': url})
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 422, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 421, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 443, in _request
    resp = self._conn.request(method, url, body=body, headers=headers)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/request.py", line 78, in request
    return self.request_encode_body(
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/connectionpool.py", line 449, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/connectionpool.py", line 444, in _make_request
    httplib_response = conn.getresponse()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/http/client.py", line 1371, in getresponse
    response.begin()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/http/client.py", line 319, in begin
    version, status, reason = self._read_status()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/http/client.py", line 280, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
KeyboardInterrupt